aixplain/modules/benchmark.py

```diff
@@ -20,12 +20,102 @@
 Description:
     Benchmark Class
 """
-from typing import List, Text
-from aixplain.modules.asset import Asset
-from aixplain.modules.benchmark_job import BenchmarkJob
-from aixplain.modules.dataset import Dataset
-from aixplain.modules.metric import Metric
-from aixplain.modules.model import Model
+import logging
+from typing import List, Text, Dict, Optional
+from aixplain.utils import config
+from aixplain.modules import Asset, Dataset, Metric, Model
```

Comment:
For cleaning reasons, split imports with `import` and `from`. If possible, sort also in alphabetical order